test/test_jit.py

```diff
@@ -3286,7 +3286,7 @@ def test_sizes(x):
             test_sizes(torch.tensor(1))
 
     def test_for_in_tensors_fail_scalar(self):
-        with self.assertRaisesRegex(RuntimeError, "cannot be used as a tuple"):
+        with self.assertRaisesRegex(RuntimeError, "cannot get length of the value type float"):
```

Comment:
These error messages are not very good. It should say something about not being iterable. What is the python error message in this case? Can we match it?